code/api/src/modules/product/query.test.js

```diff
@@ -0,0 +1,54 @@
+import request from 'supertest'
+import express from 'express'
+import schema from '../../setup/schema'
+import graphqlHTTP from 'express-graphql'
+
+describe('product query', () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+
+    server.use(
+      "/", 
+      graphqlHTTP({
+        schema: schema,
+        graphiql: true
+      })
+    )
+  })
+
+  it("is true", () => {
+    expect(true).toBe(true)
+  })
+
+  it("returns products with isSurvey: false", async () => {
+    const response = await request(server)
+    .get('/')
+    .send({
+      query: `{productSurveyTrue(productSurvey: false){
+      name
+      id
+      isSurvey
+      }}`
+    })
+    .expect(200)
+    console.log(response.body)
+    expect(response.body.data.productSurveyTrue.length).toEqual(8)
+  })
+
+  it("returns products with isSurvey: true", async () => {
+    const response = await request(server)
+      .get('/')
+      .send({
+        query: `{productSurveyTrue(productSurvey: true){
+      name
+      id
+      isSurvey
+      }}`
+      })
+      .expect(200)
+    console.log(response.body)
+    expect(response.body.data.productSurveyTrue.length).toEqual(0)
```

Comment:
We may want to try adding at least one product with isSurvey: true in this test to ensure/document that the query will return something other than 0 for the length and what it should return.

Reply:
Yes, I agree and i'll definitely change that once we figure out how we can change the isSurvey value to true.

Reply:
Sounds good, I think it won't be an issue once we merge in Melanie's seed files anyway.